Octokit.Tests/Clients/IssuesEventsClientTests.cs

```diff
@@ -26,8 +26,7 @@ public async Task RequestsCorrectUrl()
                 var client = new IssuesEventsClient(connection);
 
                 await client.GetAllForIssue("fake", "repo", 42);
-
-                connection.Received().GetAll<EventInfo>(Arg.Is<Uri>(u => u.ToString() == "repos/fake/repo/issues/42/events"), Args.ApiOptions);
+                connection.Received().GetAll<IssueEvent>(Arg.Is<Uri>(u => u.ToString() == "repos/fake/repo/issues/42/events"), null, Args.AnyAcceptHeaders, Args.ApiOptions);
```

Comment:
Can we assert that the preview URL is received here, rather than asserting that any header was found? Here's an example from elsewhere in the test suite:

https://github.com/octokit/octokit.net/blob/0074e76a8fdb00be22739bf65a1c4f48c43996e9/Octokit.Tests/Clients/DeploymentsClientTests.cs#L97-L101

We also include the preview header in the test rather than couple it to `AcceptHeader` to make preview headers more visible when reviewing.

Reply:
Done! Thanks for the feedback